src/pyfme/aircrafts/Cessna310.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jan  3 18:44:39 2016
+
+@author:olrosales@gmail.com 
+
+@AeroPython
+"""
+
+## Aircraft = Cessna 310
+
+
+import numpy as np
+
+def Geometric_Data():
+    
+    """ Provides the value of some geometric data.
+    
+    Data
+    ----
+    
+    Sw    Surface (m^2)
+    c  Mean Aerodynamic Chord (m2)
+    b    Wing Span (m2)
+    
+    References
+    ----------
+    AIRCRAFT DYNAMICS From modelling to simulation (Marcello R. Napolitano)
+    page 589
+    """
+    #falta expresar el valor de conversión entre el libro y el SI
+    Sw = 16.258
+    c = 0.4450056
+    b = 3.428122
+    
+    return Sw, c, b
+
+def Mass_and_Inertial_Data():
+    
+    """ Provides the value of some mass and inertial data.
+    
+    Data
+    -----
+    m   mass (lb * 0.453592 = kg)
+    Ixxb Moment of Inertia x-axis ( slug * ft2 * 1.3558179 = Kg * m2)
+    Iyyb Moment of Inertia y-axis ( slug * ft2 * 1.3558179 = Kg * m2)
+    Izzb Moment of Inertia z-axis ( slug * ft2 * 1.3558179 = Kg * m2)
+    Ixzb Product of Inertia xz-plane ( slug * ft2 * 1.3558179 = Kg * m2)
+    
+    References
+    ----------
+    AIRCRAFT DYNAMICS From modelling to simulation (Marcello R. Napolitano)
+    page 589
+    """
+
+    m = 4600 * 0.453592
+    Ixxb = 8884 * 1.3558179
+    Iyyb = 1939 * 1.3558179
+    Izzb = 11001 * 1.3558179
+    Ixzb = 0 * 1.3558179
+    
+    I_matrix = np.diag([Ixxb,Iyyb,Izzb])
+    
+    return m, I_matrix
+
+def Long_Aero_Coefficients():
+    
+    """Long_Aero_coefficients assigns the value of the coefficients
+    of stability in cruise conditions and order them in a matrix.
+    
+    Coefficients

```

Comment:
These `Coefficients` blocks are non-standard, I would remove the `Coefficients` header and leave the explanation of the coefficients.